app/src/pages/playground/PlaygroundInput.tsx

```diff
@@ -27,6 +27,9 @@ export function PlaygroundInput() {
         templateSyntax = "{{input name}}";
         break;
       }
+      case "NONE": {
+        return null;
+      }
       default:
```

Comment:
I'm wondering if we should just skip rendering the Input accordion entirely

<img width="1361" alt="image" src="https://github.com/user-attachments/assets/f9ab9e42-1073-4381-8dbf-2841b9553110">

What do you think @Parker-Stafford